bundles/org.openhab.core.karaf/src/main/resources/OH-INF/config/addons.xml

```diff
@@ -11,6 +11,11 @@
 			<description>Defines whether openHAB should access the remote repository for add-on installation.</description>
 			<default>true</default>
 		</parameter>
+		<parameter name="includeIncompatible" type="boolean">
```

Comment:
Shall we move this file to the `org.openhab.core` bundle? It affects all add-on services (`remote` is used by the other add-on services, too) and is IMO placed wrong in `org.openhab.core.karaf`.

Reply:
I see it is already used in the `AbstractRemoteAddonService` so that would make sense.

It might still be nice if there was also a config option to easily disable the marketplace completely. :wink:

Reply:
I added that option to the community marketplace. The JSON 3rd party add-on service can be easily disabled by deleting the URL.

Reply:
> I added that option to the community marketplace.

Yes I now see the option! :detective:

> should be shown in one place in the UI

For openHAB itself that probably makes sense. But if the other distros do not deploy some add-on service bundles, you'll probably end up with unused options in the UI. :thinking:

Reply:
I would not use the same PID, the UI should just group these PIDs in one page.

Reply:
If it can do that it would indeed be nice. IIRC the MQTT broker config that was removed in 
https://github.com/openhab/openhab-core/pull/2842 also had some kind of mechanism to group multiple configs. Though I don't know if it used different PIDs.

Reply:
I have no idea, UI is a mystery to me. Maybe @ghys can comment on that.

Anyway, I think this is out of scope here and none of the changes makes it harder to implement that later.